Simplenote/build.gradle

```diff
@@ -85,6 +92,10 @@ dependencies {
     implementation 'com.automattic:tracks:1.2'
     implementation 'org.wordpress:passcodelock:2.0.2'
 
+    // Kotlin and KTX
+    implementation 'androidx.core:core-ktx:1.3.2'
```

Comment:
Question (❓): Why did you add this library now, where will it help? I am asking that because I don't even see it being utilised on your next commit.

PS: IMHO I believe that it would be actually better if we were to be adding any libraries right when we would need them. Obviously, the `Kotlin` configuration is kind of different and thus this is okay.

Reply:
Agree. I will remove it until we use it (if we use it).

Reply:
Brilliant! 🙏